contrib/python/api/skydive/websocket/client.py

```diff
@@ -31,18 +31,18 @@
 except ImportError:
     import httplib
 import logging
-import requests
-import ssl
 import uuid
 try:
     from urllib.parse import urlparse
 except ImportError:
     from urlparse import urlparse
+import warnings
 
 from autobahn.asyncio.websocket import WebSocketClientProtocol
 from autobahn.asyncio.websocket import WebSocketClientFactory
 
 from skydive.encoder import JSONEncoder
+from skydive.rest.client import RESTClient
```

Comment:
Though it should work, do you think it is good thing to use the RESTClient here?
Maybe have a new file that both will use?
(though this is just a nit)

Reply:
yes going to move this outside in another dedicated file